packages/driver/src/cypress.js

```diff
@@ -167,6 +167,21 @@ class $Cypress {
     return this.runner.run(fn)
   }
 
+  // Method to manually re-execute Runner (usually within $autIframe)
+  // used mainly by Component Testing
+  restartRunner () {
```

Comment:
where is this used?

Reply:
@bkucera right now nowhere. In the final product in webpack-dev-server.
It is used to restart the runner when a module is "hot replaced". 
Since we chose to abandon HMR for now, I believe it can be safely removed.
@JessicaSachs @lmiller1990 should I keep this function?
How does this function work?

Reply:
As discussde today at standup, I removed the function to avoid confusion